### PR TITLE
apto-one-template: makes changes to docker files, adds makefile

### PR DESCRIPTION
**Pull request by arto80**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -45,3 +47,20 @@ services:
             - apto-mariadb-volume:/var/lib/mysql
         networks:
             - apto-docker-network
+        ports:
+            - ${APTO_DOCKER_MARIADB_PORT}:3306
+    phpmyadmin:
```
Comment (ronnybaumann) [inline docker-compose.yml]: please name service as "apto-pma-service"

File: docker-compose.yml [hunk 2]
```diff
@@ -45,3 +47,20 @@ services:
...
+        networks:
+            - apto-docker-network
+        restart: always
+        container_name: apto-phpmyadmin
```
Comment (ronnybaumann) [inline docker-compose.yml]: please name container as "apto-pma-container-${APTO_DOCKER_HOST}-${APTO_DOCKER_PORT}"

**Review by ronnybaumann — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -45,3 +47,20 @@ services:
...
         networks:
             - apto-docker-network
+        ports:
+            - ${APTO_DOCKER_MARIADB_PORT}:3306
+    apto-pma-service:
+        networks:
+            - apto-docker-network
+        restart: always
+        container_name: apto-pma-container-${APTO_DOCKER_HOST}-${APTO_DOCKER_PORT}
+        hostname: localhost
```
Comment (ronnybaumann) [inline docker-compose.yml]: shouldn't the value be "${APTO_DOCKER_HOST}" here?

**Review by ronnybaumann — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 3 above)
Comment (arto80) [reply docker-compose.yml]: no

**Pull request merged**